components/ai_chat/resources/untrusted_conversation_frame/components/conversation_entries/conversation_entries_utils.test.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { getReasoningText } from './conversation_entries_utils'
+
+describe('getReasoningText', () => {
```

Comment:
We should probably have a test for when the opening tag isn't present (and the function should probably not return anything if there isn't the matching opening tag)

Reply:
Added additional check and test https://github.com/brave/brave-core/compare/4bc169b48fb4db00b6ce6512987941271c109c1c..42de76b7aae70b4fccc029af47900d4853a6a07e